README.md

```diff
@@ -17,7 +17,7 @@ App/Models (make migration) => DB/Tables (sqlmigrate) => App/dbsqlite (migrate)
   - [ ] Understanding how the authentication/authorization stuff works in Django.
   - [ ] HTTP Basic Auth.
 - [ ] Reorganize directory structure acc to Django convention.
-- [ ] Add front-end framework(Bootstrap, UIkit, ect...) UIkit looks pretty good if we want to learn something new.
+- [X] Add front-end framework(Bootstrap, UIkit, ect...) UIkit looks pretty good if we want to learn something new.
```

Comment:
Remember, no changes to readme in a branch? :) Will introduce conflicts. 
Lets make it a convention to change it to master directly once your stuff is merged in. Then the other branch can pull in master before merge.